src/App.tsx

```diff
@@ -1,25 +1,26 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
-
 import './App.scss';
-import Start from './components/Start';
-import { Finish } from './components/Finish';
-
-import { isLoading, getMessage } from './store';
-
+import './styles/general.scss';
+import { useSelector } from 'react-redux';
+import { CurrentUser } from './components/CurrentUser';
+import { TodoList } from './components/TodoList';
+import { getUserSelector } from './store/selectors';
 
 const App = () => {
-  const loading = useSelector(isLoading);
-  const message = useSelector(getMessage) || 'Ready!';
+  const user = useSelector(getUserSelector);
```

Comment:
move user selector to CurrentUser component and make condition rendering in CurrentUser